schemas/test/en/test_calculated_summary.json

```diff
@@ -406,12 +406,14 @@
                                                         "placeholder": "total",
                                                         "transforms": [
                                                             {
-                                                                "transform": "format_number",
+                                                                "transform": "format_unit",
                                                                 "arguments": {
```

Comment:
We should update one of the placeholder schemas with an example of each.

Reply:
One of the schemas changed now to include all of these, tested in functional test, thanks!